LobbyCompatibility/Attributes/SoftLobbyCompatibilityAttribute.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using LobbyCompatibility.Enums;
+
+namespace LobbyCompatibility.Attributes;
+
+/// <summary>
+///     Specifies the compatibility of a plugin in a soft dependency manor.
+/// </summary>
+/// <example>
+///     <code>
+/// [assembly:SoftLobbyCompatibility(typeof(MyPlugin), CompatibilityLevel.ServerOnly, VersionStrictness.Minor)]
+/// 
+/// namespace ExampleMod;
+///
+/// public class MyPlugin : BaseUnityPlugin
+/// {
+/// }
+///     </code>
+/// </example>
+[AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
```

Comment:
Why AllowMultiple true?

Reply:
This is for the niche case of multiple mods being in the same assembly. It doesn't hurt to do so, as the attribute finder will only match for the first attribute that matches the plugin in question and throw away the rest (if the same plugin is registered multiple times).